### PR TITLE
feat: additional derives

### PR DESCRIPTION
**Pull request by RGafiyatullin**
Use `-i <AnotherDerive>` to additionally derive something, so that those additions do not have to be done manually after the module is generated.

### CODE REVIEW

File: src/options.rs [hunk 1]
```diff
@@ -42,6 +42,9 @@ pub struct Options {
     /// Ignore a specified column.
     #[structopt(short = "I", number_of_values = 1)]
     pub ignore: Vec<String>,
+
+    #[structopt(short = "i", number_of_values = 1)]
```
Comment (loyd) [inline src/options.rs:46]: Let's provide a comment (it will be visible in `--help`) and update README to include a new option

Comment (loyd) [inline src/options.rs:46]: Let's leave only the long form (`--derive`) here for now

**Review by loyd — CHANGES_REQUESTED** [review]

File: src/options.rs (hunk 1 above)
Comment (RGafiyatullin) [reply src/options.rs]: Done.

Comment (RGafiyatullin) [reply src/options.rs]: Done.

**Pull request merged**